src/EventStore.Core/Exceptions/UnknownFileVersionException.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace EventStore.Core.Exceptions {
+	public class UnknownFileVersionException : Exception {
```

Comment:
UnsupportedFileVersionException might be more descriptive ?

Reply:
yup, that's better thanks